moped-editor/src/views/projects/projectView/ProjectActivityLog.js

```diff
@@ -266,9 +281,11 @@ const ProjectActivityLog = () => {
                         width="80%"
                         className={classes.tableCell}
                       >
-                        {["moped_project", "moped_proj_tags"].includes(
-                          change.record_type
-                        ) ? (
+                        {[
+                          "moped_project",
+                          "moped_proj_tags",
+                          "moped_proj_funding",
+                        ].includes(change.record_type) ? (
```

Comment:
when we eventually migrate all the tables to the REST connector, we won't have to list them individually here